lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/ServiceFactory.java

```diff
@@ -125,7 +127,10 @@ private AuditService createAuditService(ObjectMapper objectMapper) {
         var commonLibConfigurationService =
                 new uk.gov.di.ipv.cri.common.library.service.ConfigurationService();
         return new AuditService(
-                SqsClient.builder().build(),
+                SqsClient.builder()
+                        .region(Region.EU_WEST_2)
+                        .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
```

Comment:
Have we confirmed this works in CRI dev? This is the change im most worried about

Reply:
This will need to be confirmed before I merge